src/containerbuddy/config_test.go

```diff
@@ -85,6 +87,15 @@ func TestValidConfigParse(t *testing.T) {
 		t.Errorf("Expected 3 args but got unexpected results: %v", args)
 	}
 
+	expectedTags := []string{"tag1", "tag2"}
```

Comment:
Can we make sure we have a test for empty tags too? Just to verify that we don't regress with it down the road?